addons/sale_management/models/sale_order_template.py

```diff
@@ -1,8 +1,9 @@
 # -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
-from odoo import api, fields, models
-from odoo.exceptions import UserError
+from odoo import api, fields, models, _
+from odoo.addons import decimal_precision as dp
```

Comment:
bad import, probably a rebase issue (decimal_precision is part of base now, no import needed)

Reply:
done.